app/src/androidTest/kotlin/com/wire/android/feature/auth/registration/CreateAccountActivityUITest.kt

```diff
@@ -12,8 +12,15 @@ class CreateAccountActivityUITest : FunctionalTest(CreateAccountActivity::class.
 
     @Test
     fun launch_uiElementsVisible() {
-        //TODO check other elements
         onView(withId(R.id.createAccountBackButton)).check(matches(isDisplayed()))
+
+        //Personal
```

Comment:
Yeah. I want to have all these tests which will serve for setting up Megazord cc @Gardosen